ports/zephyr/machine_wdt.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Paul Sokolovsky
+ * Copyright (c) 2017 Eric Poulsen
```

Comment:
I think you should definitely add your copyright line.

What I meant is that you can probably omit these existing ones.  The terms of the MIT license are "substantial portions of the Software", and I don't think there's anything of substance that was copied from another file to this one.  Unless I'm wrong?

Reply:
Ahh... somehow I understood your first comment completely different. Makes sense now. Will add it. Thanks.